src/Query.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL;
+
+use Doctrine\DBAL\Types\Type;
+
+/**
+ * A SQL query together with its bound parameters.
+ *
+ * @psalm-immutable
+ */
+class Query
+{
+    /**
+     * The SQL query.
+     *
+     * @var string
+     */
+    private $sql;
+
+    /**
+     * The parameters bound to the query, if the exception occurred during a query.
```

Comment:
The "if the exception occurred during a query" is irrelevant because otherwise this object won't be instantiated.

Reply:
Sorry, copy/paste mistake. Fixed in 05909e844c39ba011e6345dab19a9a09a8a47b27.